frontend/src/components/ShootWorkers/MachineImage.vue

```diff
@@ -23,7 +23,10 @@ SPDX-License-Identifier: Apache-2.0
         <vendor-icon :value="item.icon"></vendor-icon>
       </v-list-item-action>
       <v-list-item-content>
-        <v-list-item-title>Name: {{item.name}} | Version: {{item.version}}</v-list-item-title>
+        <v-list-item-title>
+          Name: {{item.name}} | Version: {{item.version}}
```

Comment:
we should add a tooltip, at least for the disabled items

Now we mix Labels with non-labeled items / chips. Name and version have a label, the architectures not.
I guess we have to reiterate on the design of these selects, maybe a selection from a table would be better, with having the possibility to search / filter. Probably something for a different PR